tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "target": "es6",
     "module": "ESNext",
     "moduleResolution": "node",
-    "allowSyntheticDefaultImports": false,
+    "allowSyntheticDefaultImports": true,
```

Comment:
I don't see any obvious implications of changing this configuration value. We should still be mindful about synthetic imports, as they may produce unexpected behavior in NodeJS. We must have tests that assert that.